syntax = "proto2";

message LoginRequest {
  optional string login = 1;
  optional bytes password = 2;
}
